src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+                try {
+                    testInput(input); // test todo input
+
+                    Task whatToDo = toDoMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatToDo + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+
+                    input = in.nextLine();
+                } catch (DukeExceptions e3) {
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    input = in.nextLine();
+                }
+
+
+            } else if (input.contains("deadline")) {
+                try {
+                    Task whatDeadline = deadlineMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatDeadline + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+
+                    input = in.nextLine();
+                } catch (StringIndexOutOfBoundsException e1) {
+                    System.out.println("input = " + input + "\n☹ OOPS!!! Please retype your input!");
+                    input = in.nextLine();
+                } catch (ArrayIndexOutOfBoundsException e2) {
+                    System.out.println("you have typed in = " + input + "\n☹ OOPS!!! where is your /by my friend? ");
+                    input = in.nextLine();
+                }
+
+            } else if (input.contains("event")) {
+                try {
+                    Task whatEvent = eventMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatEvent + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+                    
+                    input = in.nextLine();
+                } catch (StringIndexOutOfBoundsException e1) {
+                    System.out.println("input = " + input + "\n☹ OOPS!!! Please retype your input!");
+                    input = in.nextLine();
+                } catch (ArrayIndexOutOfBoundsException e2) {
+                    System.out.println("you have typed in = " + input + "\nw☹ OOPS!!! where is your /at brother? ");
+                    input = in.nextLine();
+                }
+
+            } else {
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");//overall error check
+                input = in.nextLine();
+
+            }
+        }
+        System.out.println("    Bye! Hope to see you again soon!");
+
     }
+
+    private static String printList(Scanner in) {
+        String input;
+        System.out.println("    Here are the tasks in your list:");
+
+        for (int outputCount = 0; outputCount < inputCount; outputCount++) {
+            System.out.println("    " + (outputCount + 1) + "." + tasks[outputCount]);
+        }
+        input = in.nextLine();
+        return input;
+    }
```

Comment:
This method exceeds 30 LOC, perhaps you could shorten it (e.g., by abstracting different operations as methods on their own).